docs/docs/50-roadmap.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Roadmap
+Description: Kargo Roadmap
+---
+
+# Roadmap
+
+Kargo does not currently have a regular release cadance. The plan is to do a few releases, and based on the resluts, create a more formal release cadance.
+
+:::caution
+This roadmap is subject to change at any time, for the most up to date information, please see the [GitHub Project](https://github.com/akuity/kargo/projects)
+:::
+
+# v0.2.0
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `Freight/Warehouse CRD` | feature | Freight will change from being a property of a `Stage`, to being it's own CRD. A collection of promotable `Freight` will be known as a `Warehouse`. |
+| `Long Lived Promotions` | feature | Support for Pull Requset based promotion, which is indefinite. This will allow the promotion to last as long as the PR is open. |
+| `Kargo Render` | breaking change | `Bookkeeper` to be retooled to be the Kargo specific way to do rendered manifests. |
+
+
+# v0.3.0
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `Analysis` | feature | Run analysis against a metrics provider to determite application health. This will allow assessing Promotion success based on more than just Argo CD Application state. |
+| `Kargo RBAC` | feature | Map SSO user identities to Kubernetes ServiceAccounts. Predefined ServiceAccount/Role/RoleBinding per project based on persona. In this version, it'll mostly be backend work. |
+| `Freight Production Rules/Filters` | feature | Optionally set up tag-based Git repository subscriptions. Additional options to produce new Freight only under certain conditions. |
+
+# v0.4.0
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `Project management` | feature | User/group/permissions management via UI and CL. Project CRD + reconciler will relieve API server of its most abusable permissions; reducing the risk profile. Additional `PromotionPolicy` options. Credential management via CLI and UI. |
+
+# v0.5.0
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `Promotion Extensibility` | feature | User defined promotion mechanisms. |
+
+# v0.6.0 - v1.0.0
```

Comment:
Nit: This looks like it will be easily misinterpreted as an intent to follow v0.6.0 with v1.0.0, when the reality is this section is "other stuff we're planning."

It's possible that misinterpretation could be discouraged like so:

```suggestion
# v0.6.0 .. v0.n.0
```